mandatory/3-function-output.js

```diff
@@ -5,7 +5,11 @@ function getRandomNumber() {
 
```

Comment:
It's worth attempting these if you get some time.
It's perfectly fine to Google something like this. We could start by searching for something like "javascript join strings together".